app/src/main/java/com/acsbendi/requestinspectorwebview/RequestInspectorJavaScriptInterface.kt

```diff
@@ -124,7 +138,8 @@ internal class RequestInspectorJavaScriptInterface(webView: WebView) {
 
     private fun addRecordedRequest(recordedRequest: RecordedRequest) {
         synchronized(recordedRequests) {
-            recordedRequests.add(recordedRequest)
+            if (!recordedRequests.equals(recordedRequest))
```

Comment:
What's the goal here? Why is it a problem if we record a request multiple times?